tests/D2L.CodeStyle.Analyzers.Test/Common/MutabilityInspectionResultFormatterTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace D2L.CodeStyle.Analyzers.Common {
+	[TestFixture]
+	class MutabilityInspectionResultFormatterTests {
+
+		private readonly MutabilityInspectionResultFormatter m_formatter = new MutabilityInspectionResultFormatter();
+
+		private static readonly IEnumerable<TestCaseData> m_testCases = new[] {
+			new TestCaseData(
+				MutabilityInspectionResult.NotMutable(),
+				string.Empty
+			).SetName("not mutable"),
+
+            // field cases

```

Comment:
I didn't test cases that are not "possible" to occur (e.g., a member that is unsealed is not possible, only a type is). I can test them, if you guys feel that's valuable.